### PR TITLE
Code Quality : If/Else refactoring and minor formattings

### PR DESCRIPTION
**Pull request by QuaintMako**
**Resolved / Related Issues**
- None

**What has been done**
- Transformed `if (...) {return ... } return...` into `return ...`.
- Coalesced some `if/else` expressions in more condensed formats
- Refactored `SortingHelper.cs`
- Inversion of some `if`s to removed nested `if`s.

**Validation**
How did you test these changes?
- [X] Built and ran the app

### CODE REVIEW

File: src/Files.App/Converters/UInt32ToString.cs [hunk 1]
```diff
@@ -7,12 +7,7 @@ internal class UInt32ToString : IValueConverter
 	{
 		public object Convert(object value, Type targetType, object parameter, string language)
 		{
-			if (value is not null)
-			{
-				return value.ToString();
-			}
-
-			return "";
+			return value is not null ? value.ToString() : "";
```
Comment (ferrariofilippo) [inline src/Files.App/Converters/UInt32ToString.cs:10]: I think you should change `""` with `string.Empty`

File: src/Files.Backend/ViewModels/Dialogs/FileSystemDialog/FileSystemDialogViewModel.cs [hunk 2]
```diff
@@ -137,29 +137,21 @@ public static FileSystemDialogViewModel GetDialogViewModel(FileSystemDialogMode
...
-						descriptionText = string.Format("ConflictingItemsDialogSubtitleSingleConflictMultipleNonConflicts".ToLocalized(), nonConflictingItems.Count);
-					}
-					else
-					{
-						// There is one conflicting file name
-						descriptionText = string.Format("ConflictingItemsDialogSubtitleSingleConflictNoNonConflicts".ToLocalized(), conflictingItems.Count);
-					}
+					// There is one conflicting file name
+					descriptionText = string.Format("ConflictingItemsDialogSubtitleSingleConflictNoNonConflicts".ToLocalized(), conflictingItems.Count);
 				}
```
Comment (ferrariofilippo) [inline src/Files.Backend/ViewModels/Dialogs/FileSystemDialog/FileSystemDialogViewModel.cs:155]: I think we should use the ternary operator here.
Something like:

**Review by ferrariofilippo — CHANGES_REQUESTED** [review]
Apart from the two small suggestionss, LGTM

File: src/Files.App/Converters/UInt32ToString.cs (hunk 1 above)
Comment (QuaintMako) [reply src/Files.App/Converters/UInt32ToString.cs:10]: Done.

File: src/Files.Backend/ViewModels/Dialogs/FileSystemDialog/FileSystemDialogViewModel.cs (hunk 2 above)
Comment (QuaintMako) [reply src/Files.Backend/ViewModels/Dialogs/FileSystemDialog/FileSystemDialogViewModel.cs]: Done.

**Review by ferrariofilippo — APPROVED** [review]
LGTM

**Comment by yaichenbaum** [comment]
Can you resolve the merge conflicts?

File: src/Files.App/DataModels/AppModel.cs [hunk 3]
```diff
@@ -109,14 +109,7 @@ private void DetectFontName()
...
 
-			if (isRunningNewIconsVersion)
-			{
-				SymbolFontFamily = new FontFamily("Segoe Fluent Icons");
-			}
-			else
-			{
-				SymbolFontFamily = new FontFamily("Segoe MDL2 Assets");
-			}
+			SymbolFontFamily = (isRunningNewIconsVersion) ? new FontFamily("Segoe Fluent Icons") : new FontFamily("Segoe MDL2 Assets");
```
Comment (yaichenbaum) [inline src/Files.App/DataModels/AppModel.cs:112]: Maybe rename `isRunningNewIconsVersion` to `isWindows11`.

Comment (QuaintMako) [reply src/Files.App/DataModels/AppModel.cs]: Done.

File: src/Files.App/Helpers/NativeWinApiHelper.cs [hunk 4]
```diff
@@ -247,17 +249,13 @@ public static bool IsHasThreadAccessPropertyPresent
...
-						Windows.Foundation.Metadata.ApiInformation.IsPropertyPresent(typeof(Windows.System.DispatcherQueue).FullName, "HasThreadAccess");
-				}
+				isHasThreadAccessPropertyPresent ??= ApiInformation.IsPropertyPresent(typeof(DispatcherQueue).FullName,"HasThreadAccess");
 				return isHasThreadAccessPropertyPresent ?? false;
 			}
 		}
 
 		// https://www.travelneil.com/wndproc-in-uwp.html
-		[ComImport, Guid("45D64A29-A63E-4CB6-B498-5781D298CB4F")]
+		[ComImport, System.Runtime.InteropServices.Guid("45D64A29-A63E-4CB6-B498-5781D298CB4F")]
```
Comment (d2dyno1) [inline src/Files.App/Helpers/NativeWinApiHelper.cs:258]: Why add namespace here?

Comment (QuaintMako) [reply src/Files.App/Helpers/NativeWinApiHelper.cs:258]: Line 250 to 254 had `Windows.Foundation.Metadata.ApiInformation` and `Windows.System` namespaces. I made them into `using` statements. 

It creates an ambiguity on the `Guid` type that could be either from `System.Runtime.InteropServices` or `Windows.Foundation.Metadata`. So it's here for disambiguation.

If we prefer having on `Guid` on l.258, I can remove the `using` statements at put it as it was previously.

**Pull request merged**